.circleci/config.yml

```diff
@@ -94,5 +94,10 @@ jobs:
             poetry run coverage xml
             /usr/local/bin/cc-test-reporter after-build -t coverage.py
 
+      - run:
+        name: Run Bandit security analysis
+          command: |
```

Comment:
Looks like an indentation error failed the build here.  You can use their CLI tool for validating the config locally before committing.

https://circleci.com/docs/2.0/local-cli/#validate-a-circleci-config